frontend/src/index.js

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import {render} from "react-dom"
+
+import "style.css"
+import {get} from "request"
+
+const Alert = () =>
+  <span style={{backgroundColor: "red", color: "white"}}>?</span>
+
+get("?json").then(data => {
+  render(
+    <div>
+      <h1>Google Scholar Alert Digest</h1>
+      <ul className="metadata">
+        <li>
+          <b>Date: </b>
+          {new Date().toUTCString()}
+        </li>
+        <li>
+          <b>Unread emails: </b>
+          <Alert />
+        </li>
+        <li>
+          <b>Paper titles: </b>
+          {data.unread.length + data.read.length}
+        </li>
+        <li>
+          <b>Unique paper titles: </b>
+          <Alert />
+        </li>
+      </ul>
+      <h2>New papers</h2>
+      <ul>
+        {data.unread.map(article => (
```

Comment:
For consistency, may be we should prefer a "paper" instead or "article", same as on the backend.
May be we could aslo introduce a single paper component (that can be in collapsed and uncollapsed states) right from the beginning, even if it is in the same file?

Reply:
Yes, I plan to refactor it into a component either way. And let's call it paper, sure.